app/src/main/java/com/framgia/soundclound/data/source/local/sqlite/AlbumLocalDataSource.java

```diff
@@ -155,6 +155,16 @@ public boolean addTrack(int idAlbum, Track track) {
         return updateAlbum(album);
     }
 
+    private boolean checkTrackExistAlbum(List<Track> tracks, Track track) {
+        for (Track trackTemp : tracks
+                ) {
```

Comment:
ERROR: 'for rparen' have incorrect indentation level 16, expected level should be 8.